android/java/org/chromium/chrome/browser/externalnav/BraveExternalNavigationHandler.java

```diff
@@ -29,7 +30,13 @@ public BraveExternalNavigationHandler(ExternalNavigationDelegate delegate) {
 
     @Override
     public OverrideUrlLoadingResult shouldOverrideUrlLoading(ExternalNavigationParams params) {
-        if (isWalletProviderOverride(params)) {
+        String originalUrl = params.getUrl().getSpec();
+        String referrerUrl = params.getReferrerUrl().getSpec();
+
+        Log.e("BraveExternalNavigationHandler",
```

Comment:
This can potentially log sensitive information.

Reply:
i am going to remove it. i added the logs for internal verification.